src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -1049,18 +1049,22 @@ string getRuleAction(DiagnosticDescriptor rule)
 
                         RulesetKind.AllDefault => getRuleActionCore(enable: rule.IsEnabledByDefault),
 
-                        RulesetKind.AllEnabled => getRuleActionCore(enable: true),
+                        RulesetKind.AllEnabled => getRuleActionCore(enable: true, enableAsWarning: true),
```

Comment:
Need a similar change for https://github.com/dotnet/roslyn-analyzers/blob/33582d0369adb7657731b611f4811e981dcb2f7f/src/Tools/GenerateDocumentationAndConfigFiles/Program.cs#L1044